packages/flutter_test/lib/src/binding.dart

```diff
@@ -106,6 +111,32 @@ mixin TestDefaultBinaryMessengerBinding on BindingBase, ServicesBinding {
   }
 }
 
+/// An accessibility announcement made by the Flutter framework.
```

Comment:
nit: this class does not represent the announcement itself, but the data captured in the test. I'd describe it as "Accessibility announcement data passed to [SemanticsService.announce] captured in a test."

Reply:
Thanks for the suggestion. I wasn't sure how to word this one.